deployment/aks-periscope.yaml

```diff
@@ -80,6 +80,8 @@ spec:
             name: kubeobjects-config
         - configMapRef:
             name: nodelogs-config
+        - configMapRef:
+            name: clustertype-config
```

Comment:
The current `master` for deployment flag has no new flag. This flag seems like the new flag : `clustertype-config` which is added as part of another refactor PR which is mostly little re-design as well.

There are 2 things which is fairly diverged here:

* If you guys 100% want to wait for refactor PR then wait because it is changing so fast that we need to understand how big as impact and what are the key decisions it needs to step back as.
* If this PR is based in Master (which seems its no longer), but in case it will be then feel free to remove it as this flag is not in master.

Key things which other PR really need to think is:

* Use of Kustomize v Helm - We need to especially keep in mind the `Cons` of helm and the security issues which comes along.
* I think the Other PR needs to be in small chunks.

Seems like there seems like there is no-needed chain of dependency created here from a work which is still in review.

Reply:
@safeermohammed @Tatsinnit  Thanks for your attention on this. I pushed up some new changes just now, which should revert all of my changes to the yaml file as discussed separately with Safeer. 
The main focus of this PR should now be on the new Helm collector I added. This new Helm collector feature has already been tested locally and with the CI pipeline. My PR also includes a unit test with tests the Helm collector feature. 
Please let me know if there's more changes that need to be made before we open this PR for other to review.